File: CursoPOOUber/Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;


class UberBlack extends Car {

    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String licence, Account driver) {
        super(licence, driver);
    }
}

File: CursoPOOUber/Python/UberBlack.py
#!/usr/bin/python3

from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []


    def __init__(self, licence, driver, typeCarAccepted, seatsMaterial):
        super().__init__(licence, driver)
        self.seatsMaterial = seatsMaterial
        self.typeCarAccepted = typeCarAccepted
